packages/backend/src/services/OrganizationService/OrganizationService.ts

```diff
@@ -204,7 +204,13 @@ export class OrganizationService extends BaseService {
         searchQuery?: string,
     ): Promise<KnexPaginatedData<OrganizationMemberProfile[]>> {
         const { organizationUuid } = user;
-        if (user.ability.cannot('view', 'OrganizationMemberProfile')) {
+
+        if (
+            user.ability.cannot(
+                'view',
+                subject('OrganizationMemberProfile', { organizationUuid }),
```

Comment:
This is redundant because organizationUuid is taken from the user, but i tis good to have this in place already if we change it in the future